app/views/partials/visualizer-settings.html

```diff
@@ -0,0 +1,310 @@
+            <option value="">Select Channel</option>
+          </select>
+          <alert ng-if="ngError.hasNoChannels"><i class="glyphicon glyphicon-warning-sign"></i>{{validationRequiredMsg}}</alert>
+          <br>
+          <table class="table table-striped table-bordered">
+            <thead>
+              <tr style="font-weight: bold">
+                <th>Channel</th>
+                <th>Display</th>
+                <th></th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr ng-repeat="channel in settings.visualizer.channels">
+                <td style="vertical-align: middle;" data-title="Channel">{{channel.eventName}}</td>
+                <td data-title="Display"><input class="form-control" type="text" ng-model="channel.display" /></td>
+                <td style="vertical-align: middle;"><button class="btn btn-danger btn-xs" ng-click="removeChannel($index)"><i class="glyphicon glyphicon-remove"></i></button></td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+      </div>
+    </div>
+
+    <div class="settings_div has-error-key">
+      <label style="float: left">
+        <span>*</span> Mediators
+        <i ng-show="uiSettings.showTooltips" tooltip-placement="top" tooltip="Select the mediators you wish to add to the visualizer. Mediators are shown just above the openhim-core." class="glyphicon glyphicon-question-sign tooltip-wide"></i>
+      </label>
+      <div class="btn-group" style="width: 100%; clear: both;">
+        <select class="form-control" ng-options="mediator as mediator.name for mediator in mediators" ng-model="visualizer.addSelectMediator" ng-change="addSelectedMediator()">
+          <option value="">Select Mediator</option>
+        </select>
+        <alert ng-if="ngError.hasNoMediators"><i class="glyphicon glyphicon-warning-sign"></i>{{validationRequiredMsg}}</alert>
+
+        <div id="no-more-tables" style="margin-top: 15px;">
+          <table class="table table-striped table-bordered">
+            <thead>
+              <tr style="font-weight: bold">
+                <th>Mediator</th>
+                <th>Display</th>
+                <th></th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr ng-repeat="mediator in settings.visualizer.mediators">
+                <td style="vertical-align: middle;" data-title="Mediator">{{mediator.name}}</td>
+                <td data-title="Display"><input class="form-control" type="text" ng-model="mediator.display" /></td>
+                <td style="vertical-align: middle;">
+                  <button class="btn btn-danger btn-xs" ng-click="removeMediator($index)"><i class="glyphicon glyphicon-remove"></i></button>
+                </td>
+              </tr>
+            </tbody>
+          </table>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+
+<div class="panel-group" id="accordion">
```

Comment:
Perhaps add a heading of 'Advanced settings' here to let the user know they don't  need to touch these.